src/views/validated-designs/index.tsx

```diff
@@ -20,26 +31,78 @@ export default function ValidatedDesignsLandingView({
 	categoryGroups,
 }: ValidatedDesignsLandingProps) {
 	return (
-		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />}>
+		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />} className={s.root}>
 			<Head>
 				<meta name="robots" content="noindex, nofollow" />
```

Comment:
let's add a title and description here.

> <title>{title}</title>
> <meta name="description" content={description} />

it's okay if they are a placeholder for now. Especially with the noindex currently.

Reply:
Added in https://github.com/hashicorp/dev-portal/commit/ae312c4f8a7ff34c28eafb1f8df5795063e45040